simpledotorg/simple-server
# Update ruby-version to 2.7.4

**Story card:** [ch3876](https://app.shortcut.com/simpledotorg/story/3876/upgrade-ruby-to-latest-2-7-release)

## Because

we want to be on the latest ruby 2

## This addresses

Upgrading ruby

## Test instructions

```
rbenv install --skip-existing 2.7.4
gem install bundler:2.2.29
bundle
rails server
```

then verify the app looks correct
